### PR TITLE
Lrouhier/update_readme

### PR DESCRIPTION
**Pull request by lrouhier**
Updates the README  following discussion about issue #13
 Add information about mounting local folder on docker container if it is not done automatically.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -154,6 +154,117 @@ Usage
...
+
+			docker-machine start
+
+		- SSH into the Docker Machine, by running command 
+
+		.. code:: sh
+
+			docker-machine ssh
+
+		- creat a new directory there by running :
```
Comment (YangHee-Min) [inline README.rst]: small spelling mistake: "creat" -> "create"

**Comment by lrouhier** [comment]
Not sure if it is the right place to add this paragraph. on the one hand the user should have already run the docker container, on the other if he has no problem it takes some space ...

**Comment by YangHee-Min** [comment]
> Not sure if it is the right place to add this paragraph. on the one hand the user should have already run the docker container, on the other if he has no problem it takes some space ...

@lrouhier Sorry not quite sure I understood what the problem was. Which paragraph are we talking about in this case?

**Comment by lrouhier** [comment]
> > Not sure if it is the right place to add this paragraph. on the one hand the user should have already run the docker container, on the other if he has no problem it takes some space ...
> 
> @lrouhier Sorry not quite sure I understood what the problem was. Which paragraph are we talking about in this case?

The one I added.

**Comment by YangHee-Min** [comment]
After following the steps (since my `docker_shared_folder` wasn't highlighted in green), I still could not manage to get the `docker_shared_folder` to be highlighted in green. I gave USERS all permissions within `C:/Users/` folder, yet cannot make a directory inside it with command `mkdir test`.

Is there something I must reset on my computer due to yesterday's tests?

**Comment by jcohenadad** [comment]
@YangHee-Min can you try to re-create another folder (different name)?

**Comment by lrouhier** [comment]
Yesterday i opened à New quickstart terminal after performing the ssh and I kept both open.  I thought it wasn't needed as you're performing ssh on the machine and running the container on top of it.

**Comment by YangHee-Min** [comment]
> @YangHee-Min can you try to re-create another folder (different name)? 

@jcohenadad  I get message: 
`mkdir: cannot create directory 'new_test': Permission denied`

**Comment by jcohenadad** [comment]
> > @YangHee-Min can you try to re-create another folder (different name)?
> 
> @jcohenadad I get message:
> `mkdir: cannot create directory 'new_test': Permission denied`

i meant: from windows (not your container), as explained [here](https://github.com/neuropoly/sct_docker#id6)

**Comment by YangHee-Min** [comment]
>>>@YangHee-Min can you try to re-create another folder (different name)?
>>
>>@jcohenadad I get message:
>>mkdir: cannot create directory 'new_test': Permission denied
>
>i meant: from windows (not your container), as explained here

Yes I was able to make the folder

**Comment by jcohenadad** [comment]
> > > > @YangHee-Min can you try to re-create another folder (different name)?
> > > 
> > > 
> > > @jcohenadad I get message:
> > > mkdir: cannot create directory 'new_test': Permission denied
> > 
> > 
> > i meant: from windows (not your container), as explained here
> 
> Yes I was able to make the folder

i know you were able to make the folder, my point was to re-test your folder sharing problem with another folder (to rule out the possibility that by playing with permissions on the same folder for the past few days you introduced some conflicts).

**Comment by YangHee-Min** [comment]
@jcohenadad  @lrouhier I retested and everything works!

**Comment by jcohenadad** [comment]
> @jcohenadad @lrouhier I retested and everything works!

do you know what went wrong during the previous test? in case a user has the same problem, we know what to tell our users...

**Comment by YangHee-Min** [comment]
>>@jcohenadad @lrouhier I retested and everything works!

>do you know what went wrong during the previous test? in case a user has the same problem, we know what to tell our users...

I hadn't properly mounted the shared folder before running docker run. In other words, I did not follow the instructions properly

**Review by jcohenadad — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -65,7 +54,13 @@ Installation
...
 #. Install `Xming <https://sourceforge.net/projects/xming/files/Xming/6.9.0.31/>`_.
 
-#. Install `Docker <https://store.docker.com/editions/community/docker-ce-desktop-windows/>`_ (or `Docker Toolbox <https://docs.docker.com/toolbox/overview/>`_ depending of your Windows version).
+#. Install Docker:
+
+   - If you are on Windows 10 Pro/Enterprise: Install `Docker <https://store.docker.com/editions/community/docker-ce-desktop-windows/>`_ 
+   
+   - If you are on Windows XP/VISTA/7/8/8.1/10 others than Pro/Enterprise: Install `Docker Toolbox <https://docs.docker.com/toolbox/overview/>`_.
+
+   See this `tutorial <https://docs.docker.com/toolbox/toolbox_install_windows/>`_ to install Docker.
```
Comment (zougloub) [inline README.rst:63]: That's only for toolbox

File: README.rst [hunk 3]
```diff
@@ -154,6 +149,117 @@ Usage
...
+#.  Check the shared folder :
+
+    - run command
+
+    .. code:: sh
+
+	ls
+
+    - the docker_shared_folder should be highlighted in green : 
+	.. image:: /picture/screenshot_green.PNG
```
Comment (zougloub) [inline README.rst:161]: Image path leading slash unneeded

File: README.rst [hunk 4]
```diff
@@ -154,6 +149,117 @@ Usage
    the changes made to it will be visible for both the Docker
    container and the Windows system.
 
+#.  Check the shared folder :
```
Comment (zougloub) [inline README.rst:152]: Double space unneeded; English colon doesn't have a space before

File: README.rst [hunk 5]
```diff
@@ -154,6 +149,117 @@ Usage
...
+
+    - run command
+
+    .. code:: sh
```
Comment (zougloub) [inline README.rst:156]: Missing indent

File: README.rst [hunk 6]
```diff
@@ -154,6 +149,117 @@ Usage
...
+
+	ls
+
+    - the docker_shared_folder should be highlighted in green :
```
Comment (zougloub) [inline README.rst:160]: Should probably have an empty line after, otherwise it may be ambiguous to the rst parser.

File: README.rst [hunk 7]
```diff
@@ -154,6 +149,117 @@ Usage
...
+
+    .. code:: sh
+
+	ls
+
+    - the docker_shared_folder should be highlighted in green : 
+	.. image:: /picture/screenshot_green.PNG
+	
+      if not : 
+	- check the permission on your local host file :
```
Comment (zougloub) [inline README.rst:164]: `-` should be aligned with `i` of `if not` above

File: README.rst [hunk 8]
```diff
@@ -154,6 +149,117 @@ Usage
...
+	    - go to C:/Users/
```
Comment (zougloub) [inline README.rst:165]: `-` should be aligned with `c` of `- check` above

File: README.rst [hunk 9]
```diff
@@ -154,6 +149,117 @@ Usage
...
+	    - Check that the USER have full control over the folder.
+
+	    .. image:: /picture/permission1.png
+
+	    - if yes move on to next paragraph.
+
+	    - if not change the permission and run the ls command again in the docker quickstart terminal. 
+	    - if the folder is highlighted in green try creating a folder inside it.
+
+	    .. code:: sh
```
Comment (zougloub) [inline README.rst:178]: missing indent

File: README.rst [hunk 10]
```diff
@@ -154,6 +149,117 @@ Usage
...
+	
+      if not : 
+	- check the permission on your local host file :
+	    - go to C:/Users/
+	    - right click on the folder open properties of the folder.
+	    - go to the security tab 
+		
+	    - Check that the USER have full control over the folder.
+
+	    .. image:: /picture/permission1.png
```
Comment (zougloub) [inline README.rst:171]: missing indent

File: README.rst [hunk 11]
```diff
@@ -154,6 +149,117 @@ Usage
...
+	    .. code:: sh
+
+		mkdir test
+
+	    - Check if a new folder appeared in C:/Users/docker_folder_shared
+	
+		
+	- Go to the docker quickstart terminal
+
+		- Stop Docker Machine if it's running, by running command:
```
Comment (zougloub) [inline README.rst:187]: Indented by too much, should be aligned with G

Comment (lrouhier) [reply README.rst:187]: You mean - Stop Docker Machine if it's running, by running command: 
should be aligned with G ?

**Pull request merged**